pkg/operation/botanist/health_check.go

```diff
@@ -290,8 +290,8 @@ func (b *Botanist) checkAPIServerAvailability(condition *gardenv1beta1.Condition
 	response, err := b.K8sShootClient.Curl("/healthz")
 	responseDurationText := fmt.Sprintf("[response_time:%dms]", time.Now().Sub(now).Nanoseconds()/time.Millisecond.Nanoseconds())
 	if err != nil {
-		message := fmt.Sprintf("Request to Shoot API server failed. %s (%s)", responseDurationText, err.Error())
-		return helper.UpdatedCondition(condition, corev1.ConditionFalse, "RequestFailed", message)
+		message := fmt.Sprintf("Request to Shoot API server /healthz endpoint failed. %s (%s)", responseDurationText, err.Error())
+		return helper.UpdatedCondition(condition, corev1.ConditionFalse, "Request to  API server /healthz endpoint failed", message)
```

Comment:
`Request to  API server /healthz endpoint failed` (condition reason) should be one word: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#typical-status-properties

How about `HealthzRequestFailed`?